src/build.js

```diff
@@ -9,7 +9,12 @@ var defaults = require('./defaults.js'),
     solveOutput = require('./solveOutput.js'),
     createCompiler = require('./createCompiler.js'),
     createLogger = require('./createLogger.js'),
-    _ = require('lodash');
+    _ = require('lodash'),
+    exitStatus = 0;
```

Comment:
It's strange anyway to listen to process exit event to then exit the process with the correct code rather than just using process.exit in the first place no?